Observer/CreditmemoRefundObserver.php

```diff
@@ -260,7 +260,8 @@ private function sendRefundRequest($memo, $orderLines, $orderObject, $payment, $
         $rawResponse = $refund->getRawResponse();
         $body        = $rawResponse->getBody();
         //add information to the altapay log
-        $this->logger->info('Response body: ', $body);
+        $xml = json_encode(simplexml_load_string($body, "SimpleXMLElement", LIBXML_NOCDATA));
```

Comment:
To get better use of error handeling:
```suggestion
        $xml = json_encode(new SimpleXMLElement($body, LIBXML_NOCDATA));
```